src/main/scala/com/databricks/labs/overwatch/pipeline/SilverTransforms.scala

```diff
@@ -1313,6 +1313,15 @@ trait SilverTransforms extends SparkSessionWrapper {
     // eagerly force this highly reused DF into cache()
     jobRunsLag30D.count()
 
+    
+    // TODO: remove or comment out or change log level or . . .
+
+    logger.log( Level.INFO, "Showing first 5 rows of `jobRunsLag30D`:")
```

Comment:
Why we r printing  5 rows of the data in logs?

Reply:
It's just to demonstrate a temporary debug output.  Any developer can use `.showLines()` anywhere & anyhow they want to during development and then remove it, comment it out, or conditionally report something at a higher log level — whatever.